rfc/rfc011-nuts-identity-credential.md

```diff
@@ -0,0 +1,136 @@
+# RFC011 Nuts Identity Credential
+
+|  |  |
+| :--- | :--- |
+| Nuts foundation | W.M. Slakhorst |
+| Request for Comments: 011 | Nedap |
+|  | February 2021 |
+
+## Nuts Identity Credential
+### Abstract
+
+A way to add identifying data to a DID.
+
+### Status
+
+This document is currently a draft.
+
+### Copyright Notice
+![](../.gitbook/assets/license.png)
+
+This document is released under the [Attribution-ShareAlike 4.0 International \(CC BY-SA 4.0\) license](https://creativecommons.org/licenses/by-sa/4.0/).
+
+## 1. Introduction
+
+Searching on DID is clumsy
+
+## 2. Terminology
+
+* **DID**: [Decentralized Identifiers (DID)](https://www.w3.org/TR/did-core/).
+* **VC**: [Verifiable Credential Data Model](https://www.w3.org/TR/vc-data-model/).
+
+## 3. Issuance
+
+```json
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://nuts.nl/credentials/v1"
+  ],
+  "id": "did:nuts:1#90382475609238467",
+  "type": ["VerifiableCredential", "NutsCompanyCredential"],
+  "issuer": "did:nuts:1",
+  "issuanceDate": "2010-01-01T19:73:24Z",
+  "expirationDate": "2010-02-01T19:73:24Z",
+  "credentialSubject": {
+    "id": "did:nuts:2",
+    "company": {
+      "tradeName": "De Nootjes",
+      "city": "Eibergen"
+    }
+  },
+  "proof": {
+    "type": "EcdsaSecp256r1Signature2019",
+    "created": "2017-06-18T21:19:10Z",
+    "proofPurpose": "assertionMethod",
+    "verificationMethod": "did:nuts:1#key-1",
+    "jws": "eyJhbGciOiJSUzI1NiIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..TCYt5X
```

Comment:
this is an RSA signature? (RS256) while the type indicates a P-256 NIST ECDSA curve?